README.md

```diff
@@ -249,6 +249,21 @@ ship.url # 'https://www.marinetraffic.com/en/ais/details/ships/shipid:434663/ves
 ship.image # 'https://i.imgur.com/woCxpkj.jpg'
 ```
 
+### Missions
+
+```ruby
+missions = SPACEX::Missions.all
```

Comment:
Please change this to either `.retrieve_all` or `.info`. Both point to the same method but `.all` doesn't exist.